BUILD

```diff
@@ -0,0 +1,30 @@
+cc_library(
+    name = "cpptrace",
+    srcs = glob([
+        "src/**/*.hpp",
+        "src/**/*.cpp",
+    ]),
+    local_defines = [
+        "CPPTRACE_GET_SYMBOLS_WITH_LIBDWARF",
```

Comment:
It's unfortunate to hard-code this, however it's pragmatic.

Reply:
Yeah, this hardcoded and unfortunate I think this is the Bazel way. Currently only LIBDWARF is supported.... Or do you think we should support all backend in one shot?